platform/android/app/src/main/kotlin/coop/polypoly/polypod/endpoint/Endpoint.kt

```diff
@@ -50,10 +87,10 @@ class Endpoint(
         observer?.approveEndpointFetch?.invoke(endpointId) {
             if (!it) {
                 logger.error("endpoint.send: User denied request")
-                throw PodApiError("")
+                throw PodApiError().endpointPermissionDenied()
```

Comment:
well, I felt it's less generic than before at least :> So we can know specifically why this error happened without looking so much the flow. Here's it's not an error of our service rather than a UI decision of the user

Reply:
Then maybe `userDeniedPermission`? Everything starts with `endPoint` and I was kinda confused here.